pkg/kubernetes/nulecule.js

```diff
@@ -85,6 +87,98 @@ define([
             return promise;
         }
 
+        function check_nulecule_version(image) {
+            var dfd = $.Deferred();
+            var msgl;
+            var versions = {};
+            debug("checking nulecule specversion");
+            var process = cockpit.spawn(["/usr/bin/atomic", "info", image])
+                .fail(function(ex) {
+                    console.warn(ex);
+                    var message = _("Unable to fetch specversion from image.");
+                    var err = new Error(message);
+                    dfd.reject(err);
+                })
+                .done(function(data) {
+                    var datal = data.split("\n");
+                    for (var i = datal.length - 1; i >= 0; i--) {
+                        if(datal[i].indexOf("io.projectatomic.nulecule.atomicappversion:") > -1) {
+                            msgl = datal[i].split('io.projectatomic.nulecule.atomicappversion:');
+                            if(msgl.length > 1)
+                                versions.atomicappversion = msgl[1];
+                        } else if (datal[i].indexOf("io.projectatomic.nulecule.specversion:") > -1) {
+                            msgl = datal[i].split('io.projectatomic.nulecule.specversion:');
+                            if(msgl.length > 1)
+                                versions.specversion = msgl[1];
+                        } else if(datal[i].indexOf("io.projectatomic.nulecule.providers:") > -1) {
+                            msgl = datal[i].split('io.projectatomic.nulecule.providers:');
+                            if(msgl.length > 1)
+                                versions.providers = msgl[1];
+                        }
+                    }
+                    debug(versions);
+                    if (Object.keys(versions).length == 3) {
+                        dfd.resolve(versions);
+                    } else {
+                        var message;
```

Comment:
Shouldn't we just say "This image is not a supported Nulecule image"? Perhaps we could log more information for people who want to debug the issue. But not show all this detail in the UI.